### PR TITLE
Add benchmarks to Docker image

### PR DESCRIPTION
**Pull request by sdww0**
This PR is related to PR #1003 and adds LMbench, Unixbench, and Iozone build scripts to the docker image.

### CODE REVIEW

**Review by StevenJiang1110 — COMMENTED** [review]
It seems the `tools/bump_version.sh` script does not bump the version of OSTD. The problem is that the `OSTD_CARGO_TOML_PATH` is not correct.

Could you please modify the line 99 in `tools/bump_version.sh` 
https://github.com/asterinas/asterinas/blob/9377a43f8fa7f8a40dbcaff602c681a99542befb/tools/bump_version.sh#L99

as

```sh
OSTD_CARGO_TOML_PATH=${ASTER_SRC_DIR}/ostd/Cargo.toml
```
and bump version again?

**Comment by sdww0** [comment]
It seems that only the OSDK CI has failed. Can we bump the OSDK version separately from the Asterinas version? This PR does not change the content of OSDK.

**Comment by tatetian** [comment]
> It seems that only the OSDK CI has failed. Can we bump the OSDK version separately from the Asterinas version? This PR does not change the content of OSDK.
> 
> Changing the `bump_version.sh` script might be helpful.

Theoretically, the versions of OSDK/OSTD and Asterinas can be different. But that complicates maintenance. Since they live under a single repository, let's keep their versions same for now.

File: tools/docker/Dockerfile.jinja [hunk 1]
```diff
@@ -138,10 +165,10 @@ FROM build-ovmf as ovmf
 WORKDIR /root
 RUN git clone --depth 1 --branch edk2-stable202402 --recurse-submodules --shallow-submodules https://github.com/tianocore/edk2.git
 WORKDIR /root/edk2
-RUN source ./edksetup.sh \ 
+RUN /bin/bash -c "source ./edksetup.sh \
```
Comment (tatetian) [inline tools/docker/Dockerfile.jinja:168]: What is the problem with the original form, which does not use `/bin/bash -c`?

**Comment by tatetian** [comment]
@sdww0 Could you fix the conflict?

File: tools/docker/Dockerfile.jinja (hunk 1 above)
Comment (sdww0) [reply tools/docker/Dockerfile.jinja:168]: When I build a docker image locally, I found that the shell cannot find ` source` without `/bin/bash -c`. I am not sure whether this situation will occur in the GitHub CI.

Comment (tatetian) [reply tools/docker/Dockerfile.jinja:168]: ok

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**